src/test/java/net/sf/jabref/importer/fileformat/SilverPlatterImporterTest.java

```diff
@@ -0,0 +1,85 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class SilverPlatterImporterTest {
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+    }
+
+    @Test
+    public final void testIsRecognizedFormat() throws Exception {
+        SilverPlatterImporter importer = new SilverPlatterImporter();
+        //test with recognized format
+        try (InputStream stream = SilverPlatterImporterTest.class
+                .getResourceAsStream("SilverPlatterImporterTest1.txt")) {
+            Assert.assertTrue(importer.isRecognizedFormat(stream));
+        }
+
+        //test with not recognized format
+        List<String> notAccepted = Arrays.asList("emptyFile.xml", "IsiImporterTest1.isi",
+                "InspecSilverPlatterImporterTest.txt", "oai2.xml", "RisImporterTest1.ris");
+        for (String s : notAccepted) {
+            try (InputStream stream = SilverPlatterImporterTest.class.getResourceAsStream(s)) {
+                Assert.assertFalse(importer.isRecognizedFormat(stream));
+            }
+        }
+    }
+
+    @Test
+    public final void testImportEntries() throws Exception {
+        SilverPlatterImporter testimporter = new SilverPlatterImporter();
+        List<BibEntry> entries = testimporter.importEntries(
+                SilverPlatterImporterTest.class.getResourceAsStream("SilverPlatterImporterTest1.txt"),
+                new OutputPrinterToNull());
+        Assert.assertEquals(17, entries.size());
+
+        entries = testimporter.importEntries(
+                SilverPlatterImporterTest.class.getResourceAsStream("SilverPlatterImporterTest2.txt"),
+                new OutputPrinterToNull());
+        Assert.assertEquals(30, entries.size());
+    }
+
+    @Test
+    public final void testImportSingleEntry() throws Exception {
+        SilverPlatterImporter testimporter = new SilverPlatterImporter();
+        for (int i = 0; i < 4; i++) {
+            try (InputStream in = SilverPlatterImporter.class.getResourceAsStream("SilverPlatterImporterTest1.txt");
+                    InputStream bibIn = SilverPlatterImporter.class
```

Comment:
As far as I can see, this test always compares `SilverPlatterImporterTest1.txt` with 4 different bib files. It is very very strange that this test passes!